pkgs/development/python-modules/minc2-simple/default.nix

```
@@ -0,0 +1,23 @@
+{ buildPythonPackage, fetchFromGitHub, stdenv, cmake,
+  hdf5, libminc, minc2_simple, minc_tools, mni_autoreg,
+  cffi, six, numpy, scipy, pytest }:
+
+buildPythonPackage {
+  pname   = "minc2-simple";
+  version = minc2_simple.version;
+
+  src = minc2_simple.src;
```

make this a little cleaner
```suggestion
  inherit (minc2_simple) src version;
```